src/plugins/intel_npu/src/compiler/src/zero_compiler_in_driver.cpp

```diff
@@ -858,19 +881,29 @@ NetworkDescription LevelZeroCompilerInDriver<TableExtension>::compile(const std:
     auto networkMeta = getNetworkMeta(graphHandle);
     networkMeta.name = model->get_friendly_name();
 
-    result = _graphDdiTableExt->pfnDestroy(graphHandle);
+    _logger.debug("compile end");
 
-    if (ZE_RESULT_SUCCESS != result) {
-        OPENVINO_THROW("Failed to compile network. L0 pfnDestroy",
-                       " result: ",
-                       ze_result_to_string(result),
-                       ", code 0x",
-                       std::hex,
-                       uint64_t(result));
+    auto networkDescription = NetworkDescription(std::move(blob), std::move(networkMeta));
```

Comment:
Blob shouldn't be needed here any longer.

Reply:
current implementation is returning an empty `std::vector<uint8_t>()`,
and go through `fillCompiledNetwork()` to get the blob for `CompiledModel::export_model` case